apps/voting/app/src/App.js

```diff
@@ -0,0 +1,107 @@
+import React from 'react'
+import styled from 'styled-components'
+import {
+  AragonApp,
+  AppBar,
+  Button,
+  SidePanel,
+  SidePanelSeparator,
+} from '@aragon/ui'
+import { votes, tokensCount } from './demo-state'
+import EmptyState from './screens/EmptyState'
+import Votings from './screens/Votings'
+import VotePanelContent from './components/VotePanelContent'
+
+class App extends React.Component {
+  state = {
+    votes,
+    createVotingVisible: false,
+    currentVoting: null,
+    votingVisible: false,
+    votingSidebarOpened: false,
+  }
+  handleCreateVoting = () => {
+    this.setState({ createVotingVisible: true })
+  }
+  handleCreateVotingClose = () => {
+    this.setState({ createVotingVisible: false })
+  }
+  handleSelectVote = id => {
+    const vote = votes.find(vote => id === vote.id)
+    if (!vote) return
+    this.setState({
+      currentVoting: vote,
+      votingVisible: true,
+      votingSidebarOpened: false,
+    })
+  }
+  handleDeselectVote = () => {
+    this.setState({ votingVisible: false })
```

Comment:
Not yet, because we want to see it during the `SidePanel` closing transition (that’s why there is `votingVisible` instead of relying on `currentVoting` only).